src/main/java/seedu/address/logic/parser/DisplayDriverTasksCommandParser.java

```diff
@@ -0,0 +1,42 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import javafx.collections.ObservableList;
+
+import seedu.address.logic.commands.DisplayDriverTasksCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.person.Driver;
+import seedu.address.model.person.DriverIsTheSamePredicate;
+
+/**
+ * Parses input arguments and creates a new DisplayDriverTasksCommand object
+ */
+public class DisplayDriverTasksCommandParser implements Parser<DisplayDriverTasksCommand> {
+
+    private static ObservableList<Driver> driverList;
+
+    public void setDriverList(ObservableList<Driver> driverList) {
+        this.driverList = driverList;
+    }
```

Comment:
is this method used anywhere? if it isn't pls remove

Reply:
yep it is used in both findDriverCommand and LogicManager.